pkg/serviceprovider/access_token_mapper.go

```diff
@@ -63,6 +66,8 @@ func (at AccessTokenMapper) ToSecretType(secretType corev1.SecretType) map[strin
 			at.Token)
 	case corev1.SecretTypeSSHAuth:
 		ret[corev1.SSHAuthPrivateKey] = at.Token
+	default:
+		ret[tokenKey] = at.Token
```

Comment:
This means that the `Opaque` secrets will ALWAYS have a field called `token`, even if there is a field mapping on the binding that would be configured differently. Are we OK with that?

Reply:
@metlos Do we have a mechanism to map it?

Reply:
@skabashnyuk I would like to understand if the double-token-field ("token" and mapped one) is the problem of Opaque token only or it may happen with any Secret type. 
For example what if we have mapping for token with SA type?
Will we see 
extra : <token>
AND
mapped-name: <token>
?

Reply:
done